components/eventing-controller/pkg/handlers/utils_test.go

```diff
@@ -198,3 +205,99 @@ func TestGetRandSuffix(t *testing.T) {
 		results[result] = true
 	}
 }
+
+func TestBebSubscriptionNameMapper(t *testing.T) {
```

Comment:
Does it make sense to test if given the same name inputs we get the same hash value?

Reply:
Please let's add a test for mapping the same input values but with different order, should result in different hash values?

For example the following inputs should give different hash value:
- foo, bar, baz => some-hash
- baz, foo, bar => different-hash

Reply:
I'm not sure I get the point. We'll be testing SHA1 basically.

Reply:
I've added a test at the end of this function. maybe it helps.